cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -18,12 +18,20 @@ import {
     Task,
     FrameSpeed,
     Rotation,
+    Workspace,
 } from 'reducers/interfaces';
 
 import getCore from 'cvat-core';
 import { RectDrawingMethod } from 'cvat-canvas';
 import { getCVATStore } from 'cvat-store';
 
+interface AnnotationsParameters {
+    filters: string[];
+    frame: number;
+    showAllInterpolationTracks: boolean;
+    jobInstance: any;
+}
+
```

Comment:
I guess it can be useful to move interface declaration to `reducers/interfaces.js`

Reply:
> @bsekachev
> 
> > @ActiveChooN
> > > Cannot edit number attribute
> > 
> > 
> > Max value for the attribute is 4.
> 
> There is no ability to change number at all. i. e. you cannot delete value to input "2" instead of "4"

Try the latest changes

Reply:
It isn't used anywhere except of this file and actually the interface is needed only to satisfy typescript static linker requirements